### PR TITLE
Feat: Homepage

### PR DESCRIPTION
**Pull request by manojncsoni007**
please review my homepage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,6 +44,69 @@
...
+                <div class="horizontal-card">
+                    <div class="card-img">
+                        <img class="card-img" src="./assets/gr5.jpg" alt="">
+                    </div>
+                    <div class="card-header flex-center">
+                        <h4 class="text-center">New Arrival</h4>
+                    </div>
+                </div>
+            </a>
+            <a href="">
```
Comment (Sharmaryan) [inline index.html:97]: href is empty, can use # as of  now

File: style.css [hunk 2]
```diff
@@ -4,7 +4,73 @@
     color: black;
```
Comment (Sharmaryan) [inline style.css:4]: use variables

File: style.css [hunk 3]
```diff
@@ -4,7 +4,73 @@
...
+    flex-wrap: wrap;
+    gap: 2rem;
+    justify-content: center;
+}
+
+.category-item {
+    width: 10rem;
+    height: 10rem;
+    margin: 0 auto;
+    border-radius: 0.8rem;
```
Comment (Sharmaryan) [inline style.css:27]: for value less than 1rem use pixels

**Review by aurobindoGupta — APPROVED** [review]
please add screen shot.
nice work

**Review by Sharmaryan — APPROVED** [review]
suggested some changes, other than that code is well written

File: style.css [hunk 4]
```diff
@@ -4,7 +4,73 @@
...
+}
+
+.category-item image {
+    width: 100%;
+    height: auto;
+    position: relative;
+}
+
+.overlay-text {
+    color: black;
```
Comment (Megha-Joshi) [inline style.css:38]: use css variables for colors

**Review by Megha-Joshi — COMMENTED** [review]
Good to merge.

**Pull request merged**